tap_github/repository_streams.py

```diff
@@ -57,17 +57,81 @@ def records_jsonpath(self) -> str:  # type: ignore
         else:
             return "$[*]"
 
+    def get_repo_ids(self, repo_list: List[Tuple[str]]) -> List[Dict[str, str]]:
```

Comment:
While we are at it, I think we should move this function to a util and add a similar one for users (and maybe orgs)

Reply:
done! I've added the same method on the `users` stream, and added a bunch of parametrized tests to try and cover all cases.
I've kept the methods in the classes as, although the structure of the code is the same, the 2 methods differ enough to not be worth trying to refactor things.